tutorials/preprocessing/plot_spatial_filter.py

```diff
@@ -0,0 +1,88 @@
+"""
+Spatial Filter
+==============
+
+This example demonstrates the effect of spatial filter on EEG data.
+"""
+
+#%%
+# .. include:: ../../../../links.inc
+
+#%%
+# .. note::
+#
+#     The lemon datasets used in this tutorial is composed of EEGLAB files. To
+#     use the MNE reader :func:`mne.io.read_raw_eeglab`, the ``pymatreader``
+#     optional dependency is required. Use the following installation method
+#     appropriate for your environment:
+#
+#     - ``pip install pymatreader``
+#     - ``conda install -c conda-forge pymatreader``
+#
+#     Note that an environment created via the `MNE installers`_ includes
+#     ``pymatreader`` by default.
+import matplotlib.pyplot as plt 
+import mne
+from mne.io import read_raw_eeglab
+
+from pycrostates.datasets import lemon
+
+raw_fname = lemon.data_path(subject_id='010004', condition='EC')
+raw = read_raw_eeglab(raw_fname, preload=True)
+
+raw.pick('eeg')
+raw.set_eeg_reference('average')
+
+#%%
+# Spatial filter was first introduced in \ :footcite:p:`michel2019eeg`
```

Comment:
Same comment as before for the footcite, I think this only includes the numerical `[1]`, and the sentence would be better with a name/year + the numerical.